packages/x-components/src/components/panels/base-header-toggle-panel.vue

```diff
@@ -32,52 +31,62 @@
    *
    * @public
    */
-  @Component({
-    components: { BaseTogglePanel }
-  })
-  export default class BaseHeaderTogglePanel extends Vue {
-    /**
-     * Animation component that will be used to animate the base-toggle-panel.
-     *
-     * @public
-     */
-    @Prop({ default: () => NoElement })
-    protected animation!: Vue | string;
-
-    /**
-     * Handles if the panel is open by default.
-     *
-     * @public
-     */
-    @Prop({ default: false })
-    protected startCollapsed!: boolean;
-
-    /**
-     * Handles if the base panel is open or closed.
-     *
-     * @internal
-     */
-    protected open = !this.startCollapsed;
-
-    /**
-     * Toggles the open property.
-     *
-     * @internal
-     */
-    protected toggleOpen(): void {
-      this.open = !this.open;
-      this.emitOpenStatusEvent();
+  export default defineComponent({
+    components: { BaseTogglePanel },
+    props: {
+      /**
+       * Animation component that will be used to animate the base-toggle-panel.
+       *
+       * @public
+       */
+      animation: {
+        type: [String, Object, Function] as PropType<string | DefineComponent>,
```

Comment:
Use here the value you created in [EMP-30](https://searchbroker.atlassian.net/browse/EMP-30)